src/functions/usdcooldown.md

```diff
@@ -4,24 +4,22 @@ description: Set a cooldown for a command
 
 # $cooldown
 
-This function sets a cooldown to the current command
+This function sets a cooldown to the current command.
 
-#### Fields
-
-This function has 2 fields
-
-1. Time \(Required\)
-2. Error \(Required\)
+### Usage 
+```php
+$cooldown[time;error]
+```
 
-Raw Usage: `$cooldown[time;error]`
+### Fields
 
-#### Options
+| Field | Description | Type | Required |
+| :--- | :--- | :--- | :--- |
+| time | The time of cooldown | alphanumeric | yes |
+| error | The error to be displayed when cooldown is active |string|yes|
 
```

Comment:
add available %options% that are present in cooldown functions

[link](https://github.com/AkaruiDevelopment/aoi.js/blob/af90c0947d41949edeb43759159aa3d1f3578e07/src/functions/Funcs/util/cooldown.js#L30)